ms_office_macro_bleach/bleach.py

```diff
@@ -60,10 +73,53 @@ def unzip_file(file):
 
 
 def remove_macros(file, notify):
+    file_type = file.split(".")[-1].lower()
+
```

Comment:
I really like this coding style, is very clean and readable :)

Reply:
Thank you :)

I used to cram everything on one line without considering readability, however with clean code principles I'm making the code more readable.